packages/headless/src/controllers/folded-result-list/headless-folded-result-list.test.ts

```diff
@@ -1,27 +0,0 @@
-import {buildFoldedResultList} from './headless-folded-result-list';
```

Comment:
Removed those unit tests for now, more unit tests are gonna be added in a future PR.